packages/authentication/src/providers/authenticate.ts

```diff
@@ -10,6 +10,12 @@ import {Strategy} from 'passport';
 import {StrategyAdapter} from '../strategy-adapter';
 import {AuthenticationBindings} from '../keys';
 
+/**
+ * Passport monkey-patches Node.js' IncomingMessage prototype
+ * and adds extra methods like "login" and "isAuthenticated"
+ */
+export type PassportRequest = ParsedRequest & Express.Request;
```

Comment:
This is my first review of authentication package :-p might be a silly question...where does the `Express` come from?

Reply:
Comes from `passport` as seen here: https://github.com/strongloop/loopback-next/pull/842/files/b6c8957f678fa2b65d5316332bda483376c95556..5fbdf0f1625aa14d30c76218379451714af402ad#diff-7eb93519e9c138afbe249e789191ee0aR9

Reply:
That doesn't explain where `Express.Request` is imported.

Reply:
I found out that `Express` is declared as a global namespace.